src/Credentials.js

```diff
@@ -16,7 +16,7 @@ class Credentials {
    *
    * @example
    * import { Credentials, SimpleSigner } from 'uport'
-   * const networks = {  '0x94365e3b': { rpcUrl: 'https://private.chain/rpc', address: '0x0101.... }}
+   * const networks = {  '0x94365e3b': { rpcUrl: 'https://private.chain/rpc', registry: '0x0101.... }}
```

Comment:
ah okay, looks like there was some confusion here because uport-lite docs uses address, but uses registry in the code. the configNetwork function here does use registry though